lib/markdown-format.coffee

```diff
@@ -2,17 +2,17 @@ markdownfmt = require('./markdownfmt.js')
 
 class MarkdownFormat
   constructor: ->
-    atom.workspace.eachEditor (editor) =>
+    atom.workspace.observeTextEditors (editor) =>
       @handleSave(editor)
 
   destroy: ->
-    atom.unsubscribe()
+    @subscriptions.dispose()
```

Comment:
Ok, I've figured it out. To make this work, you need to change it:

```diff
-@subscriptions.dispose()
+@subscription.dispose()
```

This makes _some_ sense because you're doing `@subscription = buffer.onWillSave ...` below. What makes no sense to me is why it didn't give any compilation errors...